timify-ui/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,55 @@
+import './Footer.css';
+
+const Footer = () => {
+  return (
+    <>
+      <footer className="footer">
+      <div className='footer-wrapper'></div>
+        <div className="footer-section">
+          <div>
+              <h4>Company</h4>
+              <div>
+                  <p>About Us</p>
+                  <p>Careers</p>
+                  <p>Blog</p>
+                  <p>Help</p>
+              </div>
+          </div>
+        </div>
+        <div className="footer-section">
+          <div>
+              <h4>Account</h4>
+              <div>
+                  <p>Login</p>
+                  <p>Register</p>
+                  <p>Account Setting</p>
+                  <p>My Orders</p>
+              </div>
+          </div>
+        </div>
+        <div className="footer-section">
+          <div>
+              <h4>Socials</h4>
+              <div>
+                  <p>Facebook</p>
+                  <p>Twitter</p>
+                  <p>Linkedin</p>
+                  <p>Instagram</p>
+                  <p>Youtube</p>
```

Comment:
i think the question applies here as well. what are the intention of the socials?

Reply:
same reason